kaldb/src/test/java/com/slack/kaldb/logstore/opensearch/OpenSearchQueryParserTest.java

```diff
@@ -0,0 +1,45 @@
+package com.slack.kaldb.logstore.opensearch;
+
+import static com.slack.kaldb.testlib.MessageUtil.TEST_DATASET_NAME;
+import static com.slack.kaldb.testlib.MessageUtil.makeMessageWithIndexAndTimestamp;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import brave.Tracing;
+import com.slack.kaldb.logstore.LogMessage;
+import com.slack.kaldb.logstore.search.SearchResult;
+import com.slack.kaldb.testlib.TemporaryLogStoreAndSearcherExtension;
+import java.io.IOException;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class OpenSearchQueryParserTest {
```

Comment:
This doesn't seem like it should be in a new test class - this should probably go with the other logsearcher.search behavior, in the `LuceneIndexStoreImplTest`